denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -547,11 +547,10 @@ def on_enter_zapper_function(self, event):
 
     @log_fsm_operation
     def on_enter_receive_zapper_id(self, event):
-        try:
+        zapper_api = urljoin(BASE_ZAPPER_API_URL, 'lamps/{id}?key=hash'.format(id=event.message.text))
+        response = requests.get(zapper_api)
+        if response.status_code == 200:
             line_user = LineUser.objects.get(user_id=event.source.user_id)
-        except LineUser.DoesNotExist:
```

Comment:
Why don't you catch `LineUser.DoesNotExist` exception?

Reply:
I thought the exception has never raised.

Reply:
If you run the system on another server that does not have these user data, then this exception occurs.
Although it a rare case and a misuse of this project, it would still be great that you warn the user that something terrible happens rather than let the whole system break.

Reply:
I see.
Would i print the error message by `logger.error` to indicate the missing of line user data instead of `pass`?

Reply:
Sure! That's much better!

Reply:
Done